### ISSUE TITLE
Cannot find module 'babel-runtime/core-js/promise'

### ISSUE THREAD

**Issue by Govinda-Fichtner** [jacob-meacham/serverless-plugin-git-variables#3]
Thanks for creating this useful plugin!

I tried it out yesterday by installing the node module globally and adding it to my serverless.yml.

Then I did run into the following error:
```
$ serverless --help

  Error --------------------------------------------------

     Cannot find module 'babel-runtime/core-js/promise'

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues

     Please report this error. We think it might be a bug.

  Your Environment Information -----------------------------
     OS:                 linux
     Node Version:       7.9.0
     Serverless Version: 1.6.1
```

After installing 'babel-runtime' globally, too, everything worked as expected.

I guess that 'babel-runtime' should not be a module that is necessary for just using serverless-plugin-git-variables, right?

**Comment by jacob-meacham** [comment]
Hi @Govinda-Fichtner - I just found out that I wasn't subscribed to notifications on this repository, so sorry for the slow response time! You're correct, this is definitely a bug. I'll go ahead and fix it, and publish a new version.

**Comment by jacob-meacham** [comment]
I just released v1.0.1 which fixes this issue. Thanks for filing!
